x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/alerts/basic_license_essentials_tier/alert_status/alert_status.ts

```diff
@@ -142,41 +136,13 @@ export default ({ getService }: FtrProviderContext) => {
           await waitForAlertsToBePresent(supertest, log, 10, [id]);
           const alertsOpen = await getAlertsByIds(supertest, log, [id]);
           expect(alertsOpen.hits.hits.length).equal(10);
-        });
-
-        it('should be have set the alerts in an open state initially', async () => {
-          const rule = {
-            ...getRuleForAlertTesting(['auditbeat-*']),
-            query: 'process.executable: "/usr/bin/sudo"',
-          };
-          const { id } = await createRule(supertest, log, rule);
-          await waitForRuleSuccess({ supertest, log, id });
-          await waitForAlertsToBePresent(supertest, log, 10, [id]);
-          const alertsOpen = await getAlertsByIds(supertest, log, [id]);
           const everyAlertOpen = alertsOpen.hits.hits.every(
             (hit) => hit._source?.[ALERT_WORKFLOW_STATUS] === 'open'
           );
           expect(everyAlertOpen).to.eql(true);
         });
 
-        it('should be able to close alerts while logged in and populate workflow_user', async () => {
```

Comment:
Moved to ESS specific folder due to the user we are using to test. Will need to match the test for serverless (see issues created and linked in description).